### ISSUE TITLE
Support Request: Plugin Stopped Working...

### ISSUE THREAD

**Issue by themabrennan** [OpenWonderLabs/homebridge-switchbot#576]
### Describe Your Problem

I made NO changes and the Plugin has stopped responding?!?

### Relevant log output

```shell
[07/11/2022, 11:04:41] [SwitchBot] [DEBUG] Curtain: Dining openAPIpushChanges statusCode: 200
[07/11/2022, 11:04:41] [SwitchBot] [DEBUG] Curtain: Dining Unknown statusCode.
[07/11/2022, 11:04:41] [SwitchBot] [DEBUG] Curtain: Dining d: {"statusCode":100,"body":{"items":[{"deviceID":"ED963A5E16CB","code":100,"status":{"deviceChain":2,"isCalibrate":true,"isMaster":true,"position":100,"power":"on","battery":99,"isGroup":false,"isMove":1},"message":"ok"}]},"message":"success"}
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining Refresh Status When Moving, PositionState: 0
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining openAPIRefreshStatus
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining sign: nDPMdlAHy9O4uZDG03XNCCfYuCcrO7L3+wMKGS5NGuU=
[07/11/2022, 11:04:45] [homebridge-lgwebos-tv] Device: 10.0.0.111 LG ManCave TV, debug: Socked connect error: Error: connect EHOSTUNREACH 10.0.0.111:3000, reconnect in 5s.
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining statusCode: 200
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining d: {"statusCode":100,"body":{"deviceId":"ED963A5E16CB","deviceType":"Curtain","hubDeviceId":"F7EE61FABEFB","calibrate":true,"group":false,"moving":1,"slidePosition":1},"message":"success"}
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining refreshStatus: {"statusCode":100,"body":{"deviceId":"ED963A5E16CB","deviceType":"Curtain","hubDeviceId":"F7EE61FABEFB","calibrate":true,"group":false,"moving":1,"slidePosition":1},"message":"success"}
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining openAPIparseStatus
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain Dining CurrentPosition: 99
[07/11/2022, 11:04:45] [SwitchBot] Curtain: Dining Checking Status ...
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining updateCharacteristic CurrentPosition: 99
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining updateCharacteristic PositionState: 0
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining updateCharacteristic TargetPosition: 0
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining updateCharacteristic CurrentAmbientLightLevel: 6001
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining Opening, CurrentPosition: 99 
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: 99 DECREASING PositionState: 0
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining CurrentPosition: 99, TargetPosition: 0, PositionState: 0,
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining CurrentAmbientLightLevel: 6001
[07/11/2022, 11:04:45] [SwitchBot] [DEBUG] Curtain: Dining setNewTarget true timeout
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining Refresh Status When Moving, PositionState: 0
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining openAPIRefreshStatus
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining sign: 2V4QxWXsH2/hFPLb3RGtX1gDm+liN3q4224JSvuSX9k=
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining statusCode: 200
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining d: {"statusCode":100,"body":{"deviceId":"ED963A5E16CB","deviceType":"Curtain","hubDeviceId":"F7EE61FABEFB","calibrate":true,"group":false,"moving":1,"slidePosition":1},"message":"success"}
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining refreshStatus: {"statusCode":100,"body":{"deviceId":"ED963A5E16CB","deviceType":"Curtain","hubDeviceId":"F7EE61FABEFB","calibrate":true,"group":false,"moving":1,"slidePosition":1},"message":"success"}
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining openAPIparseStatus
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain Dining CurrentPosition: 99
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining Standby, CurrentPosition: 99
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining Stopped
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining CurrentPosition: 99, TargetPosition: 99, PositionState: 2,
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining CurrentAmbientLightLevel: 6001
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining updateCharacteristic CurrentPosition: 99
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining updateCharacteristic PositionState: 2
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining updateCharacteristic TargetPosition: 99
[07/11/2022, 11:04:52] [SwitchBot] [DEBUG] Curtain: Dining updateCharacteristic CurrentAmbientLightLevel: 6001
[07/11/2022, 11:04:53] [homebridge-lgwebos-tv] Device: 10.0.0.111 LG ManCave TV, debug: Socked connect error: Error: connect EHOSTUNREACH 10.0.0.111:3000, reconnect in 5s.
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining openAPIRefreshStatus
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining sign: SJ7Cs5G0gJVwMTE2ix+zFus7zCQXTVupGixK7xJ/sAE=
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining statusCode: 200
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining d: {"statusCode":100,"body":{"deviceId":"ED963A5E16CB","deviceType":"Curtain","hubDeviceId":"F7EE61FABEFB","calibrate":true,"group":false,"moving":1,"slidePosition":1},"message":"success"}
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining refreshStatus: {"statusCode":100,"body":{"deviceId":"ED963A5E16CB","deviceType":"Curtain","hubDeviceId":"F7EE61FABEFB","calibrate":true,"group":false,"moving":1,"slidePosition":1},"message":"success"}
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining openAPIparseStatus
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain Dining CurrentPosition: 99
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining Standby, CurrentPosition: 99
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining Stopped
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining CurrentPosition: 99, TargetPosition: 99, PositionState: 2,
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining CurrentAmbientLightLevel: 6001
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining updateCharacteristic CurrentPosition: 99
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining updateCharacteristic PositionState: 2
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining updateCharacteristic TargetPosition: 99
[07/11/2022, 11:04:56] [SwitchBot] [DEBUG] Curtain: Dining updateCharacteristic CurrentAmbientLightLevel: 6001
```

### Screenshots

<img width="805" alt="Screen Shot 2022-11-07 at 10 18 27" src="https://user-images.githubusercontent.com/92501259/200371677-6c1bd128-4734-4722-ba9f-58793c8c7bd7.png">

### Device & Model

Curtains stopped responding. 

### Node.js Version

v16.18.0

### NPM Version

v1.722

### Homebridge Version

v1.5.1

### Homebridge SwitchBot Plugin Version

v2.1.2

### Operating System

Raspian

**Comment by donavanbecker** [comment]
Not sure I follow. Did you do an update or install the plugin? you said you haven't made any changes so I am assuming an update?

You may need to check on node-switchbot or noble prerequisites. 

Also you can look over [BLE Issues](https://github.com/OpenWonderLabs/homebridge-switchbot/wiki/Bluetooth-Issue) as this is tied to the install of BLE dependencies.

**Comment by themabrennan** [comment]
My apologies for not making much sense. Let me try again. 

I woke up this morning, and the HomeKit automation for the curtain did not trigger. I went into Homebridge to troubleshoot the issue. First, I downgraded the plugin with no luck. Then, I uninstalled and reinstalled the plugin and had switchbot regenerate new codes from the app. 

Unfortunately, I had the same issue where the curtains would not respond. The curtains were connected using the open API protocol and NOT BLE on Raspian.

**Comment by beatpaul** [comment]
I do have the same issue.
Switchbot curtains stopped working without doing any changes on my homebridge system on my Raspberry pi

**Comment by themabrennan** [comment]
I am getting an error in my Switchbot Bridge Logs (Screenshot attached). I happened to be browsing through the app and noticed it. 

![Screen_Shot](https://user-images.githubusercontent.com/92501259/200431661-05d40f0c-ce66-4272-9328-142c1f1b2026.jpg)

**Comment by beatpaul** [comment]
… and again same error for me in the sb app
May there be something wrong with the api?

thx for your help!

**Comment by themabrennan** [comment]
Looks like a one day bug! 

Sorry for being a pest! Everything is back working.
